src/utils/strbuf/strbuf.c

```diff
@@ -200,6 +200,17 @@ int strbuf_printf(strbuf_t *buf, char const *format, ...) {
   return 0;
 }
 
+#if !HAVE_STRNLEN
+static size_t strnlen(const char *s, size_t maxlen) {
```

Comment:
This is already in `common.c`, why is this defined again here?

Reply:
*strbuf* can't easily depend on *common*, because that would result in a circular dependency.

The cleanest solution would –imho– be to create a new package containing only the string functions, e.g. `strjoin` and friends. I didn't want to blow this change up though. How about we file that as a feature request and leave as is for now?

Reply:
Ok, sounds reasonable.